test/bin/mdx-test/expect/ocaml-errors-ellipsis/test-case.md

```diff
@@ -1,13 +1,13 @@
 It is possible to use ellipsis (`...`) in the error blocks attached to OCaml blocks, here it is useful as the error message depends on the OCaml version:
 
-```ocaml
+```ocaml version>=5.2
 module Counter: Irmin.Contents.S with type t = int64 = struct
   type t = int64
   let t = Irmin.Type.int64
 ```
 ```mdx-error
 ...
-Error: Syntax error: 'end' expected
+Error: Syntax error: end expected
 ...
```

Comment:
This test finds a new bug: A empty line is added after the last ellipsis when the block is disabled due to version.

Reply:
I don't have this bug locally. We can open another issue for later, but it shouldn't block this PR.

Reply:
This bug is reproducible with ocaml `< 5.2` and is new.

Reply:
This is now fixed in `lib/block.ml` (but I squashed everything).
Let me know what you think, but to me it looks ready to be merged.